iree/compiler/Dialect/HAL/Conversion/FlowToHAL/test/stream_ops.mlir

```diff
@@ -200,6 +241,9 @@ func @dynamic_tiled_dispatch(%arg0: tensor<7x?x24x?xf32>, %arg1: index, %arg2: i
     // CHECK-SAME: %[[COUNT_Z_INNER:.+]] = %c512 : index
 
     // This makes me so sad.
+    // If you are improving folding/canonicalization of these ops and come
+    // across this feel free to remove it all. And let me know so I can buy you
+    // a drink :)
```

Comment:
:)

Reply:
The improvement has landed in https://github.com/llvm/llvm-project/commit/039bdcc0a8a213faa8f32837c44b81ce41f41ab0, I suppose @wsmoses gets a drink :)